http_logs/operations/default.json

```diff
@@ -79,6 +79,55 @@
         }
       }
     },
+    {
+      "name": "runtime-field-source",
+      "operation-type": "search",
+      "index": "logs-*",
+      "body": {
+        "query": {
+          "match_all": {  }
+        },
+        "docvalue_fields" : ["runtime_request_is_get"],
+        "size" : 10000,
+        "_source": false
+      }
+    },
+    {
+      "name": "search-term-runtime-field",
```

Comment:
This one is covered by in the stuff I merged a few days ago with the `term` operation.